src/win32/findfile.c

```diff
@@ -91,7 +91,7 @@ static int win32_find_git_in_path(git_buf *buf, const wchar_t *gitexe, const wch
 			continue;
 		wcscpy(&root.path[root.len], gitexe);
 
-		if (_waccess(root.path, F_OK) == 0 && root.len > 5) {
+		if (_waccess(root.path, F_OK) == 0 && root.len > 5 && (root.len - 4 + wcslen(subdir) < MAX_PATH)) {
```

Comment:
Yup. We'd have avoided this issue if we simply used a `git_buf` instead, but this fix looks good to me

Reply:
Please not, this is wchar_t not char.